src/borg/archiver/prune_cmd.py

```diff
@@ -400,22 +394,20 @@ def build_parser_prune(self, subparsers, common_parser, mid_common_parser):
             "--keep-13weekly",
             dest="quarterly_13weekly",
             type=int_or_interval,
-            default=0,
             help="number or time interval of quarterly archives to keep (13 week strategy)",
         )
         quarterly_group.add_argument(
             "--keep-3monthly",
             dest="quarterly_3monthly",
             type=int_or_interval,
-            default=0,
+            default=None,
```

Comment:
So you removed `default=...` at most places, relying on `None` being the default of default.

But not here?